extension.neon

```diff
@@ -55,6 +55,8 @@ services:
 		class: PHPStan\Rules\PHPUnit\CoversHelper
 	-
 		class: PHPStan\Rules\PHPUnit\AnnotationHelper
+	-
+		class: PHPStan\Rules\PHPUnit\DataProviderHelper
```

Comment:
The rule isn't registered in rules.neon. It needs to be enabled only for bleedingEdge, feel free to get inspired by other rules that are enabled like that.

Reply:
Should be ok now